apps/test/unit/templates/studioHomepages/TeacherHomepageTest.js

```diff
@@ -0,0 +1,330 @@
+import React from 'react';
+import { assert, expect } from 'chai';
+import { mount } from 'enzyme';
+import TeacherHomepage from '@cdo/apps/templates/studioHomepages/TeacherHomepage';
+
+const announcements = [
+  {
+    heading: "first announcement heading",
+    buttonText: "first announcement buttonText",
+    description: "first announcement description",
+    link: "first announcement link"
+  },
+  {
+    heading: "second announcement heading",
+    buttonText: "second announcement buttonText",
+    description: "second announcement description",
+    link: "second announcement link"
+  }
+];
+
+const courses = [
+  {
+    name: "Course 1 name",
+    description: "Course 1 description",
+    link: "Course 1 link",
+    assignedSections: []
+  },
+  {
+    name: "Course 2 name",
+    description: "Course 2 description",
+    link: "Course 2 link",
+    assignedSections: []
+  },
+];
+
+const sections = [
+  {
+    name: "Section 1 name",
+    linkToProgress: "Section 1 link",
+    course: "Section 1 course",
+    linkToCourse: "Section 1 link to course",
+    numberOfStudents: 1,
+    linkToStudents: "Section 1 link to students",
+    sectionCode: "Section 1 section code"
```

Comment:
would prefer if our test data looked more real, i.e. links are actually links, sectionCode is something like "ASF4HE"